stable/rook-ceph/README.md

```diff
@@ -0,0 +1 @@
+See the [Operator Helm Chart](/Documentation/helm-operator.md) documentation.
```

Comment:
this file makes no sense here (also the reference is missing)

Reply:
This is unchanged from upstream. However, it is completely worthless, so I'll drop it.

Reply:
We're going to deal with this README file later.